sandbox/cgroup/cgroup.go

```diff
@@ -146,6 +148,49 @@ func ProcGroup(pid int, controller string) (string, error) {
 	return "", fmt.Errorf("cannot find cgroup controller %q path for pid %v", controller, pid)
 }
 
+// PidsInGroup returns the list of process ID currently registered in a given cgroup
+func PidsInGroup(hierarchyMount, groupPath string) ([]int, error) {
+	// TODO: check whether hierarchyMount looks like a valid cgroup root
```

Comment:
Aren't you doing that a few lines below already. If the file doesn't exist then you return. What am I missing?

Reply:
That's about adding a low tech check that the hierarchy mount point looks like a cgroup hierarchy (eg. there's at least a cgroup.procs, or maybe look at statfs). Otherwise you can pass an arbitrary path and the code will happily return nil, nil, even though you may not be looking at cgroups hierarchy at all.